src/tests/unit/tests/DetailsView/components/failure-instance-panel-control.test.tsx

```diff
@@ -94,7 +94,7 @@ describe('FailureInstancePanelControlTest', () => {
         props.instanceId = '1';
         props.editFailureInstance = editInstanceMock.object;
 
-        editInstanceMock.setup(handler => handler(description, props.test, props.step, props.instanceId)).verifiable(Times.once());
+        editInstanceMock.setup(handler => handler(description, '', '', props.test, props.step, props.instanceId)).verifiable(Times.once());
```

Comment:
Do you want to test by passing the value of `path` & `snippet` through `props`; so that if we move the `props` generation out into a separate function; this wont break this code?

Reply:
Discussed offline and since part of state instead of props in the Failure Instance Panel Control, ok to leave as hardcoded (though will abstract to a const)